app/code/Magento/Bundle/Test/Mftf/Test/MassEnableDisableBundleProductsTest.xml

```diff
@@ -130,8 +130,7 @@
         <dontSeeElement stepKey="LookingForNameOfProductDisabled" selector="{{StorefrontBundledSection.bundleProductName}}"/>
 
         <!--Enabling bundle products-->
-        <amOnPage url="{{ProductCatalogPage.url}}" stepKey="GoToCatalogPageChangingView"/>
-        <waitForPageLoad stepKey="WaitForPageToLoadFullyChangingView"/>
+        <actionGroup ref="AdminProductCatalogPageOpenActionGroup" stepKey="GoToCatalogPageChangingView"/>
```

Comment:
Lets following the camelcase format, the 1st letter should be a lowercase.
```suggestion
        <actionGroup ref="AdminProductCatalogPageOpenActionGroup" stepKey="goToCatalogPageChangingView"/>
```

Reply:
**Done**